AmericanMedicalAssociation/ama-style-guide-2
# [EWL-5119] SG2 | Resolve Backstop images for masthead

## Ticket(s)

**Jira Ticket**
- [EWL-5119: SG2 | Resolve Backstop images for masthead](https://issues.ama-assn.org/browse/EWL-5119)

## Description
As a developer, when running backstop, failures should not occur due to masthead.

## To Test

- Pull `bugfix/EWL-5119-resolve-backstop-images-masthead` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- Run `backstop test`. There should be no failures due to masthead spacing.

## Visual Regressions

Updated reference images for the event page backstop tests have been included in this PR.


## Relevant Screenshots/GIFs
N/A


## Remaining Tasks
N/A


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
